mutagen/id3/_file.py

```diff
@@ -171,6 +174,13 @@ class XMYF(Frame): ...
             else:
                 self.update_to_v24()
 
+        if self._header and combine_v1v2:
```

Comment:
Move this before translate

Reply:
That will cause issues with the precedence, since if e.g. the ID3v2 header contains a TYER tag and the ID3v1 one a TDRC tag, both will get added and then TYER removed when updating (I'll add a test for this case). `find_id3v1()` already updates the ID3v1 tags, but to be explicit I guess we can run the translation twice (before and after adding the ID3v1 tags).

Reply:
Ah, good catch. What if we make find_id3v1() take an optional v2_version arg which if 3 makes it return TYER instead of TDRC.

Reply:
Yeah, that sounds a bit cleaner